cnxepub/tests/test_collation.py

```diff
@@ -280,4 +280,4 @@ def test_with_ruleset(self):
         # Check for the appended composite document
         self.assertEqual(len(collated_binder), 3)
         self.assertEqual(collated_binder[2].metadata['title'],
-                         'Composite One')
+                         'Document One')
```

Comment:
Because they are rendered out to html, and the html renderer uses the title-override rather than the document's title, the pages will contain the correct title in both places.